src/main/java/blusunrize/immersiveengineering/common/entities/IEProjectileEntity.java

```diff
@@ -43,6 +44,7 @@
 	public boolean inGround;
 	public int ticksInGround;
 	public int ticksInAir;
+	protected IntOpenHashSet piercedEntities;
```

Comment:
This is how vanilla does it. I just didn't bother AT'ing their set, because I'd then also need to AT the list of hit entities (dunno why they have two storages).
But in style with using the most generic interface to refer to fields, I suppose I can switch this to IntSet!